processor/src/tests/stdlib/math/u64_mod.rs

```diff
@@ -157,6 +157,58 @@ fn gte_unsafe() {
     build_test!(source, &[a0, a1, b0, b1]).expect_stack(&[c]);
 }
 
+#[test]
+fn eq_unsafe() {
+    let source = "
+        use.std::math::u64
+        begin
+            exec.u64::eq_unsafe
+        end";
+
+    // a = 0, b = 0
+    build_test!(source, &[0, 0, 0, 0]).expect_stack(&[1]);
+
+    // a = 0, b = 1
+    build_test!(source, &[0, 0, 1, 0]).expect_stack(&[0]);
+
+    // a = 1, b = 0
+    build_test!(source, &[1, 0, 0, 0]).expect_stack(&[0]);
+
+    // a = 1, b = 1
+    build_test!(source, &[1, 0, 1, 0]).expect_stack(&[1]);
```

Comment:
This is basically the same as testing `a = 0, b = 0`, right? So, I'm not sure we need it.